app/src/main/java/com/sun/premierleague/ui/detail/DetailMatchActivity.kt

```diff
@@ -17,12 +21,28 @@ class DetailMatchActivity : BaseActivity(), DetailMatchContact.View {
         changeSizeWindow()
     }
 
-    override fun initComponents() {}
+    override fun initComponents() {
+        startFragmentTeam()
+    }
+
+    private fun startFragmentTeam() {
+        val getData = intent?.getStringExtra(ID_TEAM)
+        getData?.let {
+            supportFragmentManager.beginTransaction()
+                .replace(R.id.fragmentDetailMatch, TeamFragment.getInstance(it)).commit()
+        }
+    }
 
     private fun changeSizeWindow() {
         window.setFlags(
             WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS,
             WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS
         )
     }
+
+    companion object {
+        private const val ID_TEAM = "idTeam"
```

Comment:
Ở đây cũng đặt tên theo conventions nhé 
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#210-string-constants-naming-and-values
Extra phải kèm theo cả package nhé 
```suggestion
        private const val ID_TEAM = "idTeam"
```